docusaurus/docs/reactnative/customization/custom_components.mdx

```diff
@@ -107,6 +107,17 @@ const MySenderComponent = () => {
 
 This allows you as a developer to not only customize the look and feel of every part of the UI, but also decide what information is shown where based on your own design.
 
+### Hiding deleted messages from the channel
```

Comment:
"Custom Components" docs are for generic conceptual understanding of how to customize Chat SDK components. I think this example is better suited for Message Customization docs - https://getstream.io/chat/docs/sdk/reactnative/guides/message-customization/